app/models/metadata_presenter/service.rb

```diff
@@ -36,8 +44,29 @@ def confirmation_page
     end
   end
 
+  def meta
+    MetadataPresenter::Meta.new(configuration['meta'])
+  end
+
   private
 
+  def all_pages
+    @all_pages ||= pages + standalone_pages
+  end
```

Comment:
So note to ourselves: more urls we need to restrict later (all meta pages plus the restricted ones)

Reply:
have added it to the card for doing that work